parking/templates/index.html

```diff
@@ -37,7 +37,7 @@
         <table class="table" id="myTable">
             <thead>
                 <tr>
-                    <th scope="col">Car Number</th>
```

Comment:
**Changes to be made:**

1. Use snake cases for variable names and function names and pascal casing for classes.
2. Remove print statements before pushing your code to github.
3. You can use single form for create and update actions if possible.
4. Try to avoid same naming for project and apps.
5. I think it would be better if you create a new model for handling car parking information. By doing this you can add some more fields related to car parking like parking timestamps (when was the car parked , what time did it leave )